newIDE/app/src/AnnouncementsFeed/index.js

```diff
@@ -65,6 +67,7 @@ export const AnnouncementsFeed = ({
     title: string,
     message: string,
     isMarkdownImageOnly?: boolean,
+    onInternalClick?: () => void,
   |} => {
```

Comment:
I would almost move this into a standalone function outside of the component and unit test it, as the regex logic is growing a bit complex.

Reply:
good idea